invenio_i18n/translations/da/LC_MESSAGES/messages.po

```diff
@@ -22,4 +22,4 @@ msgstr ""
 #: invenio_i18n/templates/invenio_i18n/macros/language_selector.html:31
 #: invenio_i18n/templates/invenio_i18n/macros/language_selector.html:41
 msgid "Language:"
-msgstr ""
+msgstr "Sprog:"
```

Comment:
Have you pushed it to Transifex? Nobody should update PO files locally, but instead run `tx pull -l da` after editing online.

--
See https://www.transifex.com/inveniosoftware/invenio/translate/#da/invenio-i18n-messages/95780369